auto-complete.js

```diff
@@ -62,6 +62,44 @@
             });
         }
 
+
+    window.changeText = () => {
+            
+        const newText = 
+             `
+             <div style="margin: 0.4em" class="row">
+                <span style="font-size: 10px" type="button" class="chip chip--outline chip--sm" data-toggle="tooltip" data-placement="bottom-right"
+                    data-tooltip ="Identifique as palavras ou termos que, obrigatoriamente, estejam na sua pesquisa.
+                    Exemplo: Direitos E Humanos">
+                    E
+                </span>
+                <span style="font-size: 10px" type="button" class="chip chip--outline chip--sm" data-toggle="tooltip" data-placement="bottom-right"
+                    data-tooltip ="Identifique palavras ou termos para obter resultados com pelo menos uma das palavras-chave especificadas. 
+                    É possível utilizar parênteses para agrupar frases.Exemplo: (Dano moral) OU (Recurso Especial)" >
+                    OU
+                </span>
+                <span style="font-size: 12px" type="button" class="chip chip--outline chip--sm" data-toggle="tooltip" data-placement="bottom-right"
+                    data-tooltip ="Nenhum dos resultados conterão o(s) termo(s)excludentes, indicados após o NÃO.Exemplo: (dano moral) NÃO material" >
+                    Não Incluir
+                </span>
+                <span style="font-size: 12px" type="button" class="chip chip--outline chip--sm" data-toggle="tooltip" data-placement="bottom-right"
+                    data-tooltip ="Os resultados conterão os termos na ordem exata e com a exata grafia indicada.Exemplo: princípio da presunção de inocência">
+                    Exatamente
+                </span>
+            </div> `  
+
+        document.getElementById('search').innerHTML = newText;           
```

Comment:
Avoid using `id`, this won't work if there is 2 or more elements in the same page with same `id`. Also, `search` is too common name to be used for `id` and likely to conflict.
```suggestion
        this.parentElement.innerHTML = newText;           
```